pkg/model/lattice/rule.go

```diff
@@ -32,11 +32,12 @@ type RuleSpec struct {
 	PathMatchValue  string `json:"pathmatchvalue"`
 	PathMatchExact  bool   `json:"pathmatchexact"`
 	PathMatchPrefix bool   `json:"pathmatchprefix"`
-	// Header
+	// HeaderMatch
 	NumOfHeaderMatches int `json:"numofheadermatches"`
 	MatchedHeaders     [MAX_NUM_OF_MATCHED_HEADERS]vpclattice.HeaderMatch
-	// Method
-	Method     string     `json:"method"`
+	// MethodMatch
+	Method string `json:"method"`
```

Comment:
nit: can we align properties, remove empty lines between them. comments can be added at the end of the line

Reply:
Sure thing